service/src/common/email-template-payload/community.external.invitation.created.email.payload.ts

```diff
@@ -0,0 +1,14 @@
+import { BaseJourneyEmailPayload } from './base.journey.email.payload';
+
+export interface CommunityExternalInvitationCreatedEmailPayload
+  extends BaseJourneyEmailPayload {
+  inviter: {
+    name: string;
+    firstName: string;
+    email: string;
+    profile: string;
+  };
+  welcomeMessage?: string;
+  emails?: string;
```

Comment:
Why `emails` is string and not string[] ?

Reply:
This is the email template payload, it can accept only strings.